server/src/main/java/org/elasticsearch/index/mapper/NumberFieldMapper.java

```diff
@@ -1032,7 +1086,14 @@ public abstract Query rangeQuery(
 
         public abstract Number parsePoint(byte[] value);
 
-        public abstract List<Field> createFields(String name, Number value, boolean indexed, boolean docValued, boolean stored);
+        public abstract void createFields(
```

Comment:
I wonder if this should be renamed to `addFields` now that it no longer just instantiates the objects?

Also, would be nice to add javadoc on the expected behavior.

Reply:
++ Renamed and added javadoc.